graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -1065,11 +1097,7 @@ def ast_to_ir(schema, ast, type_equivalence_hints=None):
                                 *****
 
     Returns:
-        IrAndMetadata named tuple, containing fields:
-        - ir_blocks: a list of IR basic block objects
-        - input_metadata: a dict of expected input parameters (string) -> inferred GraphQL type
-        - output_metadata: a dict of output name (string) -> OutputMetadata object
-        - query_metadata_table: a QueryMetadataTable object containing location metadata
+        IrAndMetadata for the given schema/AST
```

Comment:
nit: thoughts on having an explicit `and` instead of the `/`? I'm a bit worried it might get misinterpreted as an `or`.